File: C#/BluffinMuffin.Protocol.DataTypes/TupleTable.cs
using System;
using BluffinMuffin.Protocol.DataTypes.Attributes;
using BluffinMuffin.Protocol.DataTypes.Enums;

namespace BluffinMuffin.Protocol.DataTypes
{
    /// <summary>
    /// 
    /// </summary>
    public class TupleTable : IComparable<TupleTable>
    {
        /// <summary>
        /// The id of the table
        /// </summary>
        [ExampleValue(42)]
        public int IdTable { get; set; }
        /// <summary>
        /// The nb of players currently sitting at the table
        /// </summary>
        [ExampleValue(6)]
        public int NbPlayers { get; set; }
        /// <summary>
        /// The action available to you for this table (Nothing, Join, Leave)
        /// </summary>
        [ExampleValue(LobbyActionEnum.Join)]
        public LobbyActionEnum PossibleAction { get; set; }
        /// <summary>
        /// Parameters of the poker table
        /// </summary>
        public TableParams Params { get; set; }

        /// <summary>
        /// 
        /// </summary>
        /// <param name="other"></param>
        /// <returns></returns>
        public int CompareTo(TupleTable other)
        {
            return IdTable.CompareTo(other.IdTable);
        }
    }
}

File: Python/BluffinMuffin.Protocol/data_types/tuple_table.py
from data_types.enums.game_type_enum import GameTypeEnum
from data_types.enums.lobby_action_enum import LobbyActionEnum
from data_types.table_params import TableParams

__author__ = 'ericmas001@gmail.com'



class TupleTable:
    def __init__(self, obj):
        self.id_table = obj['IdTable']
        self.nb_players = obj['NbPlayers']
        self.possible_action = LobbyActionEnum.parse(obj['PossibleAction'])
        self.params = TableParams(obj['Params'])

    def __str__(self):
        return '{0}({1}) {2} [{3}]'.format(
            self.id_table,
            self.nb_players,
            LobbyActionEnum.to_string(self.possible_action),
            self.params
        )
